packages/hub/schema.graphql

```diff
@@ -594,6 +611,22 @@ type RelativeValuesPairCache implements Node {
   secondItem: String!
 }
 
+type SearchEdge {
+  cursor: String!
+  node: Searchable!
+  rank: Float!
+  slugSnippet: String!
+  textSnippet: String!
+}
+
+type Searchable implements Node {
+  id: ID!
+  link: String!
```

Comment:
`link` field is awkward because we could have `link` field on all specific searchable objects, e.g. `User.link` and `Model.link` would be useful.

We can implement those later and deprecate this field.

Reply:
Yea, I'm not too worried. Could always use a different name.

I guess we could change this to ``searchLink`` if we want, but can easily do that later too.

Reply:
I didn't mean there's any problem with the name, just meant that because `link` field is useful on object level (one level deeper), it's weird to duplicate it here.